src/libs/ajax/AzureStorage.ts

```diff
@@ -165,7 +170,8 @@ export const AzureStorage = (signal?: AbortSignal) => ({
     };
 
     return {
-      getData: getObjectMetadata,
+      getMetadata: () => getBlobByUri('HEAD'),
+      getData: () => getBlobByUri('GET'),
```

Comment:
TOL could this be `getMetadataAndTextContent`? Is that name now getting obnoxious?

Reply:
Not too obnoxious, I don't have a strong opinion!